implementation/interfacing_with_emlib.tex

```diff
@@ -164,32 +164,226 @@ \subsection{Naming Conventions}
 \label{lst:enum_naming_rust}
 \end{listing}
 
+\subsection{The Bindings Libraries}
```

Comment:
Ja, du mener at `The Binding Libraries` burde komme før `Defining the Bindings`?